var/spack/repos/builtin/packages/neurodamus3/package.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack import *
+from spack.pkg.builtin.neurodamus_model import NeurodamusModel
+
+
+class Neurodamus3(NeurodamusModel):
```

Comment:
As i mentioned before, I think the name is a bit confusing. I would leave it as Neurodamus

Reply:
Agree with Jorge - what `3` indicates here?